tabular/src/autogluon/tabular/trainer/abstract_trainer.py

```diff
@@ -336,13 +337,24 @@ def stack_new_level_core(self, X, y, models: List[AbstractModel], X_val=None, y_
                     return []
             else:
                 raise AssertionError(f'Stack level cannot be negative! level = {level}')
-            models = [
-                stacker_type(
-                    path=self.path, name=model.name + '_STACKER_l' + str(level), model_base=model, base_model_names=base_model_names,
+            invalid_model_names = set(self.get_model_names_all())
```

Comment:
What about use-case where somebody does want to overwrite an existing model?

Reply:
added TODO for v0.1, don't want to touch args for these methods until TabTransformer PR is merged to avoid conflicts